resources/home/dnanexus/generate_workbook/utils/excel.py

```
@@ -599,7 +599,7 @@ def atlas_summary(self) -> None:
         row_count = 9
 
         # write counts of variants
-        for sheet, vcf in zip(self.args.sheets, self.vcfs, strict=True):
+        for sheet, vcf in zip(self.args.sheets, self.vcfs):
```

Ruff (0.16.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)